lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -58,6 +58,7 @@ def __init__(self, runbook: schema.Platform) -> None:
         self.libvirt_conn_str: str
         self.qemu_platform_runbook: QemuPlatformSchema
         self.host_node: Node
+        self.vm_disks_dir: str
```

Comment:
The static typing can be specified on first assignment, line Ln 281 in this case. Only if the first place uses the variable, it needs to declare like this way as a workaround.

Reply:
Adding it here makes life easier for the IDE language servers, which makes the IDE language integration more reliable.